internal/core/api.rs

```diff
@@ -489,11 +489,11 @@ impl Window {
             }
             crate::platform::WindowEvent::Resized { size } => {
                 self.0.set_window_item_geometry(size.to_euclid());
-                self.0
-                    .window_adapter()
-                    .renderer()
-                    .resize(size.to_physical(self.scale_factor()))
-                    .unwrap()
+                if let Ok(non_zero_size) = size.to_physical(self.scale_factor()).try_into() {
```

Comment:
Is it really invalid to have a window size of zero?

For example, imagine a window embedded in another window (a frame)  and it is resized to its minimum size by the user so that it is zero.  

I think IMHO a zero size is a valid size, and that we should resize the renderer appropriately.
Then the renderer should know that there is nothing to render and could destroy the surface or do whatever. But I feel we should still pass an actual size to the renderer::resize.
But since this is a private API, it doesn't matter.